src/main/java/org/ossgang/commons/mapbackeds/Mapbackeds.java

```diff
@@ -157,23 +245,40 @@ public M build() {
     }
 
     public static Set<Method> fieldMethods(Class<?> intfc) {
+        return collectIntfcMethods(intfc, m -> !m.isDefault());
+    }
+
+    public static Optional<Method> toStringMethod(Class<?> intfc) {
+        Set<Method> toStringMethods = toStringMethods(intfc);
+        if (toStringMethods.size() > 1) {
+            throw new IllegalArgumentException(
+                    "More than one method with a ToString annotation found in the hierarchy!");
+        }
+        return toStringMethods.stream().findFirst();
+    }
+
+    private static Set<Method> toStringMethods(Class<?> intfc) {
+        return collectIntfcMethods(intfc, m -> m.isAnnotationPresent(ToString.class));
+    }
+
+    private static Set<Method> collectIntfcMethods(Class<?> intfc, Predicate<Method> methodPredicate) {
```

Comment:
Probably I would write out `Interface` in the method name here - it's just 4 characters more and a little more clear ;-)

Reply:
Done